src/style.css

```
@@ -0,0 +1,113 @@
+html,
+body,
+form,
+img,
+div {
+  margin: 0;
+}
+
+@font-face {
+  font-family: "AvenirBook";
+  font-style: normal;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "AvenirBold";
+  font-style: normal;
+  font-weight: 400;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.lg-search-bar__form,
+.sm-search-bar__form {
+  position: relative;
+  width: 100%;
+  margin-top: 20px;
+  margin-left: 8px;
+}
```

To make the input full width, remove this width: 100%. Forms have `display: block` and will take all width anyway. Having `widht: 100%` and `margin-left` will make it wider than the screen.
```suggestion
.lg-search-bar__form,
.sm-search-bar__form {
  position: relative;
  margin-top: 20px;
  margin-left: 8px;
}
```